<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KernelToolWidget</class>
 <widget class="ToolWidget" name="KernelToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KernelToolWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_layer" stretch="10">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_layer">
       <item>
        <widget class="QLabel" name="label_layer">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Input Point Features' Layer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_layer"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_field" stretch="10">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_fiedl">
       <item>
        <widget class="QLabel" name="label_field">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Choose Population Field</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_field"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_dist" stretch="10">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_dist">
       <item>
        <widget class="QLabel" name="label_out_2">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Distance Option</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_dist"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_cell">
     <item>
      <widget class="QLabel" name="label_cell">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Output Cell Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_cell">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Please input cell size</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_radius">
     <item>
      <widget class="QLabel" name="label_output_radius">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search Radius</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_radius">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Please input search radius</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_btn">
     <property name="spacing">
      <number>40</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_default">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ToolWidget</class>
   <extends>QWidget</extends>
   <header>toolwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
